test/runtests.jl

```diff
@@ -180,7 +180,7 @@ AstroTime.update()
         leap = UTCEpoch(2016, 12, 31, 23, 59, 60)
         tai1, tai2 = ERFA.utctai(julian1(leap), julian2(leap))
         @test Epochs.utctai(julian1(leap), julian2(leap)) == ERFA.utctai(julian1(leap), julian2(leap))
-        @test_broken Epochs.taiutc(tai1, tai2) == ERFA.taiutc(tai1, tai2)
+        @test Epochs.taiutc(tai1, tai2) == ERFA.taiutc(tai1, tai2)
```

Comment:
Since the strict equality test is failing on 32bits again I would propose to move all ported function tests to approximate equality.

Reply:
all means all the test till now or only this?